lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -467,15 +467,12 @@ sub setup_vm_simple_dns_with_ip {
 
     my $_dns_file = '/etc/hosts';
 
-    # Workaround for directly editing file issue: resource busy
-    if (is_alp) {
-        $_dns_file = '/etc/hosts.wip';
-        assert_script_run "cp /etc/hosts $_dns_file";
-    }
-
     script_run "sed -i '/$_vm/d' $_dns_file";
     assert_script_run "echo '$_ip $_vm' >> $_dns_file";
-    assert_script_run "cp $_dns_file /etc/hosts" if (is_alp);
+    my $cmd = qq(nmcli dev show | grep DOMAIN | awk '{print \$2}' | uniq);
```

Comment:
This is the handling with network manager. Please also keep the previous way for earlier wicked, by if else. This file is shared with MU tests too, which have sle15/sle12 tests without NM.